README.md

```
@@ -45,6 +45,7 @@ This new vision for Archon replaces the old one (the agenteer). Archon used to b
 - [Docker Desktop](https://www.docker.com/products/docker-desktop/)
 - [Supabase](https://supabase.com/) account (free tier or local Supabase both work)
 - [OpenAI API key](https://platform.openai.com/api-keys) (Gemini and Ollama are supported too!)
+- [Make](https://www.gnu.org/software/make/) (see [Installing Make](#installing-make) below)
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid

(MD051, link-fragments)